brands/amenity/cinema.json

```
@@ -295,8 +339,34 @@
       "name": "Haut et Court"
     }
   },
+  "amenity/cinema|Les Écrans de Paris": {
+    "locationSet": {
+      "include": ["fr"]
+    },
+    "tags": {
+      "amenity": "cinema",
+      "brand": "Les Écrans de Paris",
+      "brand:wikidata": "Q3236510",
+      "brand:wikipedia": "fr:Les Écrans de Paris",
+      "name": "Les Écrans de Paris"
+    }
+  },
+  "amenity/cinema|Groupe Majestic Cinemas": {
```

Also just use the name "Majestic" here too
```suggestion
  "amenity/cinema|Majestic": {
```

---

Okay, I fixed them. Making new PR